haystack/modeling/data_handler/processor.py

```diff
@@ -625,6 +631,12 @@ def _convert_answers(self, baskets: List[SampleBasket]):
                 else:
                     # For all other cases we use start and end token indices, that are relative to the passage
                     for i, answer in enumerate(basket.raw["answers"]):
+                        if i >= self.max_answers:
+                            logger.warning(
+                                f"Found a sample with more answers ({len(basket.raw['answers'])}) than "
+                                f"max_answers ({self.max_answers}). These will be ignored."
```

Comment:
Pylint is complaining about using f-strings in logging, we should use % formatting instead:
```suggestion
                                "Found a sample with more answers (%d) than "
                                "max_answers (%d). These will be ignored." % (len(basket.raw['answers']), self.max_answers)
```